apiary.apib

```diff
@@ -3777,6 +3779,7 @@ Update a draft work order.
         + materials (array, optional)
             + (object)
                 + quantity: `2.2` (number, required)
+                + purchase_price (Money, optional)
```

Comment:
Q: This is only `optional` and not `optional, nullable` on purpose, right?

Reply:
But it's a collection, we just overwrite the collection completely in updates, right? 🤔